src/core/models.py

```diff
@@ -230,6 +236,28 @@ def create_superuser(self, email, password, **kwargs):
     def get_queryset(self):
         return AccountQuerySet(self.model)
 
+    def get_or_create(self, defaults=None, **kwargs):
+        """
+        Backwards-compatible override for affiliation-related kwargs
+        """
```

Comment:
we should instead do this at the Queryset level, since django will delegate this call to `AccountQuerySet` and the queryset also exposes these ORM operations.

Also, don't forget about `get`, `create`, `filter` and `update_or_create`

The bulk operations can be ignored IMO